tests/unit/unified-resolver/basic-dynamic-test.js

```diff
@@ -34,7 +34,7 @@ function expectResolutions({ message, config, resolutions, moduleOverrides, erro
   let fakeRegistry = new NewFakeRegistry({
     moduleOverrides
   });
-  let resolver = Resolver.create({
+  let resolver = new Resolver({
```

Comment:
I believe `create` is the invocation signature used [by ember](https://github.com/emberjs/ember.js/blob/0f96d18ee895cfcfe880aaa91a2b170ff06a3891/packages/ember-application/lib/system/engine.js#L438). In general instantiating Ember objects with `new` and an argument is not really considered supported public API.